src/shelly_main.c

```diff
@@ -304,14 +304,16 @@ static void shelly_get_info_handler(struct mg_rpc_request_info *ri,
       ri,
       "{id: %Q, app: %Q, host: %Q, version: %Q, fw_build: %Q, uptime: %d, "
 #ifdef MGOS_CONFIG_HAVE_SW1
-      "sw1: {id: %d, name: %Q, in_mode: %d, persist: %B, state: %B"
+      "sw1: {id: %d, name: %Q, in_mode: %d, persist: %B, state: %B, auto_off: "
+      "%B, auto_off_delay: %d}"
```

Comment:
there shouldn't be closing brace at the end here

Reply:
"sw1: {id: %d, name: %Q, in_mode: %d, persist: %B, state: %B, "
      "auto_off: %B, auto_off_delay: %d"

Reply:
Oops! Good find, fixing...